app/assets/javascripts/listings/templates/listing/_panel-apply.html.slim

```diff
@@ -91,7 +91,9 @@
 
   .content-group.bg-snow
     p.c-charcoal.t-small
-      | Do not apply online and also send in a paper application. All applications containing any person who appears on more than one application will be removed from the lottery.
+      span ng-if="listing.Accepting_Online_Applications && !listing.Lottery_Results"
```

Comment:
@callaghanc is the `&& !listing.Lottery_Results` just to ensure that this listing is still open? I don't think that part of the `ng-if` is necessary because the whole `_panel-apply` section will already be hidden once the listing is closed.